pkg/apis/ignite/v1alpha1/conversion.go

```diff
@@ -174,3 +174,9 @@ func Convert_v1alpha1_VMKernelSpec_To_ignite_VMKernelSpec(in *VMKernelSpec, out
 
 	return Convert_v1alpha1_OCIClaim_To_ignite_OCI(&in.OCIClaim, &out.OCI)
 }
+
+// Convert_v1alpha1_VMNetworkSpec_To_ignite_VMNetworkSpec calls the autogenerated conversion function and custom conversion logic
+func Convert_v1alpha1_VMNetworkSpec_To_ignite_VMNetworkSpec(in *VMNetworkSpec, out *ignite.VMNetworkSpec, s conversion.Scope) error {
+	// .Spec.Network.Mode is not tracked in the v1alpha2 and newer, so there's no extra conversion logic
```

Comment:
".spec.network.mode is not tracked in v1alpha2 and newer, so there's no extra conversion logic"

Reply:
`.Spec.Network.Mode` is exactly as valid, even more IMO. `.spec.network.mode` is the JSON path